lib/xcode.js

```diff
@@ -243,9 +243,6 @@ exports.detect = function detect(options, callback) {
 			globalSimRuntimes = findSimRuntimes('/Library/Developer/CoreSimulator/Profiles/Runtimes'),
 			xcodes = [];
 
-		// since we do not support Xcode 5 and below, weed them out
-		options.supportedVersions = (options.supportedVersions ? options.supportedVersions + '||' : '') + '>=6.0.0';
```

Comment:
You can't remove this. We actually use this: https://github.com/appcelerator/titanium_mobile/blob/7505a2937424157b38ad87744cfbb05e8424dba4/iphone/cli/commands/_build.js.

The problem is the `||`. We need the equivalent of a `&&`, but semver does not support `&&`.

However, there's an easy workaround. Just set a `&&` in the `options.supportedVersions`, then where `supportedVersions` is referenced, call your own `satisfies(<VERSION>, options.supportedVersions)`.

```js
function satisfies(ver, ranges) {
    if (!ranges) {
        return true;
    }

    return ranges.split('&&').every(function (range) {
        return appc.version.satisfies(ver, range, true);
    });
}
```

Reply:
@cb1kenobi I can't remember where we landed after chatting, are we fine with taking the risk of someone having xcode 5 installed on their machine?

Reply:
Last we chatted, we were talking about bumping this to v1.8.0 and creating a 1_8_X branch being that this is a breaking change. I don't think this is a bad idea.

I also was thinking we should add a blockquote in the readme making note that ioslib only supports Xcode 6 or newer.

Reply:
As rediscussed, this doesn't remove support for 6 but just removes the hardcoding of >=6 support (which was added to avoid blowing up on Xcode 5), the note was already added https://github.com/appcelerator/ioslib/blame/1_7_X/README.md#L7. It's ready whenever you are I guess :)

Reply:
Well then, rock n roll!